src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1217,6 +1222,35 @@ public static void applySslContext()
         }
     }
 
+    public static void applyCryptoProvider()
+    {
+        if (conf.crypto_provider == null)
+            conf.crypto_provider = new ParameterizedClass(DefaultCryptoProvider.class.getName(), null);
+
+        if (conf.crypto_provider.parameters == null)
+            conf.crypto_provider.parameters = new HashMap<>();
+
+        if (conf.crypto_provider.class_name == null)
+            throw new ConfigurationException("Failed to initialize crypto provider, class_name cannot be null");
+
+        conf.crypto_provider.parameters.putIfAbsent(AbstractCryptoProvider.FAIL_ON_MISSING_PROVIDER_KEY, "false");
+
+        try
+        {
+            Class<?> cryptoProviderClass = Class.forName(conf.crypto_provider.class_name);
```

Comment:
I think we can use `FBUtilities#classForName` here.
OR
alternatively use `FBUtilities#construct(java.lang.String, java.lang.String)` and pass the `ParameterizedClass` instance from the `config` directly to the `install` method.

Reply:
classForName is fine